var/spack/repos/builtin/packages/py-black/package.py

```diff
@@ -18,6 +18,8 @@ class PyBlack(PythonPackage):
     version('19.3b0', sha256='68950ffd4d9169716bcb8719a56c07a2f4485354fec061cdd5910aa07369731c')
     version('18.9b0', sha256='e030a9a28f542debc08acceb273f228ac422798e5215ba2a791a6ddeaaca22a5')
 
+    variant('d', default=False, description='enable blackd HTTP server')
```

Comment:
Why specify a single-character variant name here?  It doesn't make the variant very intuitive.

Reply:
That's how the pip variant is named. If you want a blackd server, you run `pip install black+d`